src/main/cc/proxy_client/javac_remote_actions.cc

```diff
@@ -11,15 +15,74 @@ const int kMaxArgsToCheckForJavac = 8;
 const char* kJavacWrapperCommand = "javac_wrapper ";
 const char* kJavacCommand = "javac ";
 
+const char* kJavacWithOptionsCommand = "javac -";
+
 using std::min;
+using std::string;
+using std::set;
+using std::cout;
+
+bool IsAlphabet(const char c) {
```

Comment:
I suspect there's a function in basl for this.

Reply:
Done.